website/src/parsers/vue/vue-eslint-parser.js

```diff
@@ -18,6 +18,9 @@ export default {
   },
 
   parse(parser, code, options) {
+    if (Object.keys(options).length === 0) {
+      options = this.getDefaultOptions();
+    }
```

Comment:
I don't think this is strictly necessary (the default options are always passed), but there might be some edge case I'm not considering. Did you add this because you ran into issues?

Reply:
I added this because I had the same issue as acorn.

https://github.com/fkling/astexplorer/pull/588#discussion_r647355376

Reply:
I see. I doesn't hurt to add this, I just hope it won't be necessary in the future :)